src/test/java/core/basesyntax/handler/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class SupplyOperationHandlerTest {
+    private static Fruit apple;
+    private static OperationHandler supplyOperationHandler;
+    private final FruitTransaction fruitTransaction = new FruitTransaction();
+
+    @BeforeAll
+    static void setUp() {
+        apple = new Fruit("apple");
+        supplyOperationHandler = new SupplyOperationHandler();
+    }
+
+    @BeforeEach
+    void clearStorage() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    void appendProductBySupplyHandler_Ok() {
+        fruitTransaction.setOperation(FruitTransaction.Operation.SUPPLY);
+        fruitTransaction.setQuantity(33);
+        fruitTransaction.setFruit(apple);
+        supplyOperationHandler.apply(fruitTransaction);
+        Integer expected = 33;
+        Integer actual = Storage.storage.get(apple);
+        Assertions.assertEquals(expected, actual);
+    }
```

Comment:
add a test so that there is already some of the fruit in Storage and they will deliver more